{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.0",
	"cveMetadata": {
		"cveId": "CVE-2022-4770",
		"assignerOrgId": "dce6e192-ff49-4263-9134-f0beccb9bc13",
		"assignerShortName": "HITVAN",
		"state": "PUBLISHED",
		"datePublished": "2023-04-03T18:56:17.800Z",
		"dateUpdated": "2023-04-03T18:56:17.800Z",
		"dateReserved": "2022-12-27T22:39:50.860Z"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "dce6e192-ff49-4263-9134-f0beccb9bc13",
				"shortName": "HITVAN",
				"dateUpdated": "2023-04-03T18:56:17.800Z"
			},
			"title": "Hitachi Vantara Pentaho Business Analytics Server - Generation of Error Message Containing Sensitive Information ",
			"descriptions": [
				{
					"lang": "en",
					"value": "\nHitachi Vantara Pentaho Business Analytics Server prior to versions 9.4.0.0 and 9.3.0.2, including 8.3.x display the full parametrized SQL query in an error message when an invalid character is used within a Pentaho Report (*.prpt).Â \n\n",
					"supportingMedia": [
						{
							"type": "text/html",
							"base64": false,
							"value": "\n\n<span style=\"background-color: rgb(255, 255, 255);\">Hitachi Vantara Pentaho Business Analytics Server prior to versions 9.4.0.0 and 9.3.0.2, including 8.3.x display the full parametrized SQL query in an error message when an invalid character is used within a Pentaho Report (*.prpt).</span><span style=\"background-color: rgb(255, 255, 255);\">&nbsp;</span>\n\n"
						}
					]
				}
			],
			"affected": [
				{
					"vendor": "Hitachi Vantara",
					"product": "Pentaho Business Analytics Server",
					"defaultStatus": "unaffected",
					"versions": [
						{
							"status": "affected",
							"versionType": "maven",
							"version": "1.0",
							"lessThan": "9.3.0.2"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "CWE",
							"lang": "en",
							"description": "CWE-209 Generation of Error Message Containing Sensitive Information",
							"cweId": "CWE-209"
						}
					]
				}
			],
			"impacts": [
				{
					"descriptions": [
						{
							"lang": "en",
							"value": "CAPEC-54 Query System for Information"
						}
					],
					"capecId": "CAPEC-54"
				}
			],
			"metrics": [
				{
					"format": "CVSS",
					"scenarios": [
						{
							"lang": "en",
							"value": "GENERAL"
						}
					],
					"cvssV3_1": {
						"version": "3.1",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N",
						"attackVector": "NETWORK",
						"attackComplexity": "LOW",
						"privilegesRequired": "LOW",
						"userInteraction": "NONE",
						"scope": "UNCHANGED",
						"confidentialityImpact": "LOW",
						"integrityImpact": "NONE",
						"availabilityImpact": "NONE",
						"baseScore": 4.3,
						"baseSeverity": "MEDIUM"
					}
				}
			],
			"references": [
				{
					"url": "https://support.pentaho.com/hc/en-us/articles/14455209015949--Resolved-Hitachi-Vantara-Pentaho-Business-Analytics-Server-Generation-of-Error-Message-Containing-Sensitive-Information-Versions-before-9-4-0-0-and-9-3-0-2-including-8-3-x-Impacted-CVE-2022-4770-"
				}
			],
			"credits": [
				{
					"type": "finder",
					"lang": "en",
					"user": "00000000-0000-4000-9000-000000000000",
					"value": "Hitachi Group Member"
				}
			],
			"source": {
				"discovery": "INTERNAL"
			},
			"x_generator": {
				"engine": "Vulnogram 0.1.0-dev"
			}
		}
	}
}
